jujugui/static/gui/src/app/components/deployment-flow/expert-intro/expert-intro.js

```diff
@@ -65,7 +65,8 @@ class DeploymentExpertIntro extends React.Component {
       const entity = entityModel.toEntity();
       const machineNumber = this.isBundle ? entity.machineCount : 1;
       const price = `From $${entity.price} per month`;
-      const supportedDescription = marked(entity.supportedDescription);
+      const supportedDescription = entity.supportedDescription ?
+        marked(entity.supportedDescription) : null;
```

Comment:
We should probably put a message here like "The author has not provided a description." or something along those lines.

Reply:
Done and test added.